packages/spartan/extra/dcos-spartan.service

```diff
@@ -19,6 +19,6 @@ ExecStartPre=/usr/bin/env ip link set spartan up
 ExecStartPre=-/usr/bin/env ip addr add 198.51.100.1/32 dev spartan
 ExecStartPre=-/usr/bin/env ip addr add 198.51.100.2/32 dev spartan
 ExecStartPre=-/usr/bin/env ip addr add 198.51.100.3/32 dev spartan
-ExecStart=/opt/mesosphere/active/spartan/spartan/bin/spartan-env foreground
+ExecStart=/usr/bin/env flock -x /opt/mesosphere/active/spartan/spartan/bin/spartan-env /opt/mesosphere/active/spartan/spartan/bin/spartan-env foreground
```

Comment:
@sargun can you give us a timeline on what is happening around this `spartan-env` file and how this fixes the watchdog startup issue? This flock approach implies that something is trying to write to this file, and nothing should be writing to `/opt/mesosphere/`...

What does your approach to fix this with dcos-spartan-watchdog and an ExecStarPre step look like? Fixing the issue there seems to make more sense to me intuitively.